tests/interface/test_tracing.py

```diff
@@ -7,9 +7,7 @@ def test_tracing_v0_interface(interface_tester: InterfaceTester):
     interface_tester.configure(
         interface_name="tracing",
         interface_version=0,
-        # todo replace with upstream CRI when
-        #  https://github.com/canonical/charm-relation-interfaces/pull/41 lands
-        repo="https://github.com/PietroPasotti/charm-relation-interfaces",
-        branch="tracing",
+        repo="https://github.com/canonical/charm-relation-interfaces",
+        branch="main",
```

Comment:
Interface test is failing now:
```
ERROR    interface_tester.interface_test:runner.py:55 Failed check 3: validating schema on scene output: 2 validation errors for ProviderSchema
app -> url
  field required (type=value_error.missing)
app -> ingesters -> 0 -> protocol
  field required (type=value_error.missing)
```
In https://github.com/canonical/charm-relation-interfaces/pull/92, the filename `interfaces/tracing/v0/interface_tests/requirer_tests.py` wasn't renamed to `proiver_tests`. Does it matter?

@simskij @PietroPasotti